datastore/mongo/organisation_member_test.go

```diff
@@ -200,17 +275,35 @@ func TestFetchOrganisationMemberByID(t *testing.T) {
 	require.Equal(t, m.UID, member.UID)
 	require.Equal(t, m.OrganisationID, member.OrganisationID)
 	require.Equal(t, m.UserID, member.UserID)
+	require.Equal(t, datastore.UserMetadata{
+		UserID:    user.UID,
+		FirstName: user.FirstName,
+		LastName:  user.LastName,
+		Email:     user.Email,
+	}, *member.UserMetadata)
 }
 
 func TestFetchOrganisationMemberByUserID(t *testing.T) {
-	db, closeFn := getDB(t)
-	defer closeFn()
+	db, _ := getDB(t)
+	//defer closeFn()
```

Comment:
Why is this line commented out? You're also ignoring the second value from the `getDB()` function

Reply:
Missed that, i used it for testing. Fixed now.